src/shared/security/htmlSanitizer.ts

```diff
@@ -0,0 +1,133 @@
+import DOMPurify from 'isomorphic-dompurify';
```

Comment:
consider barrel index export for this directory and possibly one level up in shared